commit b6edd41c86d391dea3bd866b4ef65deb41405937
Author: Hubert Figuiere <hub@figuiere.net>
Date:   Sun May 26 08:53:14 2024 +0000

    poppler/Catalog: call getNumPages() only once in findPage()

diff --git a/poppler/Catalog.cc b/poppler/Catalog.cc
index 211197a4..488c61fd 100644
--- a/poppler/Catalog.cc
+++ b/poppler/Catalog.cc
@@ -346,9 +346,9 @@ bool Catalog::cachePageTree(int page)
 
 int Catalog::findPage(const Ref pageRef)
 {
-    int i;
+    const int count = getNumPages();
 
-    for (i = 0; i < getNumPages(); ++i) {
+    for (int i = 0; i < count; ++i) {
         Ref *ref = getPageRef(i + 1);
         if (ref != nullptr && *ref == pageRef) {
             return i + 1;
